src/frontend/.eslintrc.json

```diff
@@ -28,11 +26,13 @@
     }],
     "flowtype/sort-keys": [
       "error",
-      "asc", {
-        "caseSensitive": true,
-        "natural": false
-      }
-    ]
+      "asc"
+    ],
+    "array-callback-return": 0,
```

Comment:
Rules from this line and below were added to make the linter less mad. I think we should open an issue to remove some or all of the following rules, and actually make the corresponding code changes, since I think some of these rules are actually legit and should not be ignored. Any suggestions?

Reply:
Agreed. I created an issue here: https://github.com/azavea/echo-locator/issues/483